config/application.rb

```diff
@@ -108,5 +108,8 @@ class Application < Rails::Application
 
     config.public_prisons_with_slot_availability =
       ENV['PUBLIC_PRISONS_WITH_SLOT_AVAILABILITY']&.split(',')&.map(&:strip) || []
+
+    config.staff_prisons_with_nomis_contact_list =
+      ENV['STAFF_PRISONS_WITH_NOMIS_CONTACT_LIST']&.split(',')&.map(&:strip) || []
```

Comment:
There seem to be an emerging pattern here, a encapsulating this behaviour in a united tested class might not have been a bad idea.